mos-platform/geos-cbm/clang.cfg

```diff
@@ -0,0 +1,8 @@
+-Os
+-flto
+-fnonreentrant
```

Comment:
OK, I see you have enabled nonreentrant whereby you may ignore my previous comment on `__attribute__((leaf))` on inline asm.

Reply:
This really isn't an okay thing to have by default in a `clang.cfg` file. It really dramatically  changes the semantics of C globally, and users should have to opt in themselves.

Reply:
Removed it from clang.cfg file.